### PR TITLE
chore: [asset swapper] sampler Solidity 0.6 + Bridge addresses in AS

### PR DESCRIPTION
**Pull request by dekz**
## Description

Upgrades the sampler to solidity 0.6.
Refactors into try/catch where possible

[Sell sims](https://metabase.spaceship.0x.org/dashboard/139?run_id=choke-obtainable-scissors&adjusted_amount_bought_win_tolerance__bps_=0.5)
* 0x
* Kyber
* Balancer
* Mooniswap
* DODO
* Uniswap_V2
* SushiSwap
* Swerve
* Curve
* LiquidityProvider
* MultiHop

Missing:
* MStable
* Eth2Dai
* Shell

Included in price comparisons, just not good pricing at the moment
```js
{
name: "Shell",
price: "0.980418",
gas: "300000"
},
{
name: "Eth2Dai",
price: "0.968364",
gas: "400000"
},
```

```
{
name: "mStable",
price: "0.9994",
gas: "700000"
},
```

## Checklist:

-   [ ] Prefix PR title with `[WIP]` if necessary.
-   [ ] Add tests to cover changes as needed.
-   [ ] Update documentation as needed.
-   [ ] Add new entries to the relevant CHANGELOG.jsons.

### CODE REVIEW

File: packages/asset-swapper/contracts/src/LiquidityProviderSampler.sol [hunk 1]
```diff
@@ -136,14 +130,15 @@ contract LiquidityProviderSampler is
         if (registryAddress == address(0)) {
             return address(0);
         }
+
         bytes memory callData = abi.encodeWithSelector(
-            ILiquidityProviderRegistry(0).getLiquidityProviderForMarket.selector,
+        ILiquidityProviderRegistry(0).getLiquidityProviderForMarket.selector,
```
Comment (dorothy-zbornak) [inline packages/asset-swapper/contracts/src/LiquidityProviderSampler.sol:135]:
```suggestion
        ILiquidityProviderRegistry.getLiquidityProviderForMarket.selector,
```

File: packages/asset-swapper/contracts/src/LiquidityProviderSampler.sol [hunk 2]
```diff
@@ -136,14 +130,15 @@ contract LiquidityProviderSampler is
...
             takerToken,
             makerToken
         );
         (bool didSucceed, bytes memory returnData) = registryAddress.staticcall(callData);
```
Comment (dorothy-zbornak) [inline packages/asset-swapper/contracts/src/LiquidityProviderSampler.sol:139]: Why no `try/catch` here?

**Review by dorothy-zbornak — APPROVED** [review]
So much cleaner. Thank you!

**Review by mzhu25 — APPROVED** [review]
thanks!

File: packages/asset-swapper/contracts/src/LiquidityProviderSampler.sol (hunk 2 above)
Comment (dekz) [reply packages/asset-swapper/contracts/src/LiquidityProviderSampler.sol:139]: There's explicit tests for when the registry doesn't exist, where we want to fail silently. I believe we hit the case where the abi decode failed and reverted.

**Comment by romainbutteaud** [comment]
@dekz I think you forgot SnowSwap GAS_SCHEDULE?

**Pull request merged**